docs/sdk/v3/guides/advanced/03-price-oracle.md

```diff
@@ -168,15 +168,13 @@ We cannot directly use the value of a single Observation for anything meaningful
 const diffTickCumulative = observations[0].tickCumulative - observations[1].tickCumulative
 const secondsBetween = 108
 
-const averageTick = diffTickCumulative / secondsBetween
+const averageTick = Number(diffTickCumulative / BigInt(secondsBetween))
```

Comment:
Is this allowed?

Reply:
Yes, and the result of this calculation will always be representable by a Number